runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1415,19 +1489,21 @@ func s:GotoAsmwinOrCreateIt()
       exe 'new'
     endif
 
-    let s:asmwin = win_getid(winnr())
+    let s:asmwin = win_getid()
 
     setlocal nowrap
     setlocal number
     setlocal noswapfile
     setlocal buftype=nofile
+    setlocal bufhidden=wipe
```

Comment:
These now being `bufhidden=wipe` is sus, as now if you `:q` one of the windows, you get E86 from `s:asmbuf` or `s:varbuf` being invalid. I'll see what I can do about it upstream.